### ISSUE TITLE
Add requirements.txt install note?

### ISSUE THREAD

**Issue by daguar** [codeforamerica/US-Census-Area-API#3]
As someone who's not a Pythonista, the step missing in the instructions for me was:

``pip install -r requirements.txt`

I was going to do a PR, but I wanted to check whether you want this, or would prefer to point to the (soon to come) VirtualEnv tutorial.

**Comment by migurski** [comment]
You're right, the tutorial (which is now public!) is the right way to go. PR away!

**Comment by migurski** [comment]
https://github.com/codeforamerica/howto/blob/master/Python-Virtualenv.md

(also, interesting that you’re “installing” it—I had imagined it would mostly be used via Heroku only)

**Comment by daguar** [comment]
Well it was for the "test locally" piece.

I feel like this is actually a really interesting case study for "dev experience" in the README.

For example, it seems like more or less like a "cloud appliance" - to that end, maybe the top bit of the README should be an as-simple-as-humanly-possible "add shape, commit, and push" kind of instructions, with local setup and other stuff way below.

It's sort of "Appliance Quick Deploy" instructions up top, the cloud equivalent of a QuickStart.

The idea to me is that an appliance is opinionated intentionally to "just work." I as a dev shouldn't even care how it works until/unless it doesn't solve my problem.

I might take a stab at what this would look like to me if I have a bit of time and motivation.

**Comment by migurski** [comment]
Yes—what about authenticating to Heroku and creating an app there on behalf of the user?

**Comment by migurski** [comment]
Oh, one tricky part: getting the GDAL library into Python is non-trivial, as it requires compiled binaries. Keeping this appliance-style to the point of recommending a VM might be wise.

**Comment by ErnieAtLYD** [comment]
As a fellow of a different year taking a stab at this, I got hung up also trying to install this locally as well and was recommended to install this through heroku. (Although I'm actually still having some problems getting the zipped file URL within _heroku-buildpack-pygeo_ working properly...)
